src/App.js

```diff
@@ -6,18 +6,25 @@ import { fetchBusinesses } from './services/yelp';
 function App() {
   const [businesses, setBusinesses] = useState([]);
   const [loading, setLoading] = useState(true);
+  const [zip, setZip] = useState('97202');
+  const [search, setSearch] = useState('');
 
-  // TODO -- add state for zip / search and add event listeners to the inputs
+  // TODO -- add state for zip / search and add event listeners to the inputs --CHECK
 
   useEffect(() => {
     const fetchData = async () => {
-      const data = await fetchBusinesses();
-      setBusinesses(data);
+      const resp = await fetchBusinesses();
+      setBusinesses(resp.businesses);
       setLoading(false);
+      //setBusiness(resp.businesses)
     };
     fetchData();
   }, []);
 
+  const handleClick = async () => {
+    const data = await fetchBusinesses(zip, search);
+    return setBusinesses(data.businesses);
```

Comment:
We will also want to handle if the user does not add in a zip code before we make our fetch call, as an empty zip code will cause the app to break.